apps/artemis-web/pages/admin/system.tsx

```diff
@@ -63,34 +64,38 @@ const SystemPage = (props) => {
     exabgptap: 'ExaBGP Monitor',
     detection: 'Detection',
     mitigation: 'Mitigation',
+    database: 'Database',
   };
 
   modules.forEach((module) => (states[module[0].toString()] = module[1]));
-  const [state, setState] = useState(states);
-  const keys = Object.keys(state).filter((key) =>
+  // const [state, setState] = useState(states);
+  let stateRT = states;
```

Comment:
stateRT means real-time state? Two options:
1. Either use clearer names
2. Comment on what it means so that the reader knows how it is used in the rest of the code